src/einsteinpy/symbolic/metric.py

```diff
@@ -108,3 +108,21 @@ def lower_config(self):
         if self.config == "ll":
             return self
         return self.inv()
+
+    def lorentz_transform(self, transformation_matrix):
+        """
+        Performs a Lorentz transform on the tensor.
+
+        Parameters
+        ----------
+            transformation_matrix : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+                Sympy Array or multi-dimensional list containing Sympy Expressions
+
+        Returns
+        -------
+            ~einsteinpy.symbolic.tensor.MetricTensor
+                lorentz transformed tensor(or vector)
+
```

Comment:
`~einsteinpy.symbolic.metric.MetricTensor` due to the reasons stated previously.
remove `or vector`